composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
The point of PSR-3 is that you don't need a specific logger. If anything, we should require monolog via equip/project. But not here.